src/backend/XmlHttpRequest.ts

```diff
@@ -69,11 +75,12 @@ export default class XmlHttpRequest extends BaseFileSystem implements FileSystem
     if (typeof(opts.index) === "string") {
       XmlHttpRequest.FromURL(opts.index, cb, opts.baseUrl, false);
     } else {
-      cb(null, new XmlHttpRequest(opts.index, opts.baseUrl, false));
+      cb(null, new XmlHttpRequest(opts.index, opts.baseUrl, false, opts.use));
```

Comment:
[For context, this is XHR in the V2 branch](https://github.com/jvilk/BrowserFS/blob/dropbox_v2/src/backend/XmlHttpRequest.ts#L88). 

(I won't make you rebase; I can handle the merge into V2 myself.)